torch/fx/passes/splitter_base.py

```diff
@@ -63,6 +64,7 @@ def __init__(self):
 
 
 # TODO: this can probably be optimized
+@compatibility(is_backward_compatible=False)
```

Comment:
cc @842974287 FYI I'm conservatively setting backward-compatibility for the APIs in the file to `False`. Please feel free to flip this to `True` if/when you feel comfortable providing stability guarantees for the API.